ui/src/components/Publication/Visualization/index.tsx

```diff
@@ -17,7 +17,7 @@ interface BoxEntry {
     authorFirstName: string;
     authorLastName: string;
     publishedDate: string;
-    authors: Pick<Interfaces.CoAuthor, 'id' | 'linkedUser' | 'publicationId' | 'user'>[];
+    authors: Pick<Interfaces.CoAuthor, 'id' | 'linkedUser' | 'publicationVersionId' | 'user'>[];
```

Comment:
I have reworked Visualisation in #493 . Will need to handle some conflicts in this area either in that PR or in this one depending on which is merged first.